tests/docs/test_examples.py

```diff
@@ -15,14 +15,17 @@
 os.environ["SMOKE_TEST"] = "true"
 
 
+paths = [str(x) for x in Path("examples/").rglob("*.py")]
+
+
 @pytest.mark.slow
 @pytest.mark.skipif(
     not (_CHEM_INSTALLED and _ONNX_INSTALLED), reason="skipped for core tests"
 )
-@pytest.mark.parametrize("example", Path("examples/").rglob("*.py"))
-def test_example(example: Path):
+@pytest.mark.parametrize("example", paths, ids=paths)
```

Comment:
Ah, clever 🤓 Makes sense that there is the option to pass an "ids" list. I wasn't as smart in an earlier PR, see `test_docs.py`. Could you perhaps also just quickly fix that file as well?

Reply:
keep in mind that I had to chagne from Path's to strings for that, if thats stillf ien for you a corresponding commit has been added

Reply:
fighting hard against my internal desire to have 100% type-safety, but accept :D